apis/proto/v1/gateway/vald/vald.proto

```diff
@@ -87,7 +87,8 @@ service Vald {
     option (google.api.http).delete = "/remove/{id}";
 
   }
-  rpc StreamRemove(stream payload.v1.Object.ID) returns (stream payload.v1.Object.Location) {}
+  rpc StreamRemove(stream payload.v1.Object.ID) 
+      returns (stream payload.v1.Object.Location) {}
```

Comment:
StreamResponses is not used in vald.proto?

Reply:
I'm thinking about it, should i fix v1/gateway/vald/vald.proto???
may be gateway/vald will be duplicated I think we shouldn't fix anything about it... may be...:)

Reply:
maybe you mean "deprecated". 
i see. okay.